Apps/Common/src/Services/AuditService.cs

```diff
@@ -19,16 +19,30 @@ namespace HealthGateway.Common.Services
     using System.Threading.Tasks;
     using Microsoft.Extensions.Configuration;
     using HealthGateway.Common.Database.Models;
+    using Microsoft.AspNetCore.Http;
+    using System.Reflection;
+    using Microsoft.Extensions.Primitives;
+    using System.Net;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Routing;
 
     /// <summary>
     /// The Authorization service
     /// </summary>
     public class AuditService : IAuditService
     {
+        private const string ANONYMOUS = "anonymous";
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AuditService"/> class.
+        /// </summary>
+        /// <param name="config">The injected configuration provider.</param>
         public AuditService(IConfiguration config)
         {
 
         }
+
+        /// <inheritdoc />
         public Task WriteAuditEvent(AuditEvent auditEvent)
         {
             Task t = Task.Factory.StartNew(() => 
```

Comment:
Nit: single letter names

Reply:
not touching this code since brad is going to implement it